cmd/launcher/svc_config_windows.go

```diff
@@ -184,3 +187,56 @@ func checkRestartActions(logger *slog.Logger) {
 
 	logger.Log(context.TODO(), slog.LevelInfo, "successfully set RecoveryActionsOnNonCrashFailures flag")
 }
+
+// setRecoveryActions sets the recovery actions for the launcher service.
+// previously defined via wix ServicConfig Element (Util Extension) https://wixtoolset.org/docs/v3/xsd/util/serviceconfig/
+func setRecoveryActions(_ context.Context, logger *slog.Logger) {
+	sman, err := mgr.Connect()
+	if err != nil {
+		logger.Log(context.TODO(), slog.LevelError,
```

Comment:
Might as well use the context passed in for these log messages? Or remove it from the args if we don't need it?

Reply:
oops, now using context passed